tests/test_rewards_manager.py

```diff
@@ -1,21 +1,28 @@
 import pytest
-from brownie import reverts, ZERO_ADDRESS, chain, accounts
-from utils.config import lido_dao_agent_address, rewards_amount
+from brownie import reverts, ZERO_ADDRESS, chain, accounts, RewardsManager
+from utils.config import lido_dao_agent_address, rewards_amount, gift_index
 
 
 def test_owner_is_deployer(rewards_manager, ape):
     assert rewards_manager.owner() == ape
 
 
+def test_rewards_contract_address_is_not_zero(ape):
+    with reverts("zero address"):
+        rewards_manager_ = RewardsManager.deploy(ZERO_ADDRESS, {"from": ape})
+
+
 def test_stranger_cannot_transfer_ownership(rewards_manager, stranger):
     with reverts("not permitted"):
         rewards_manager.transfer_ownership(stranger, {"from": stranger})
 
 
 def test_ownership_can_be_transferred(rewards_manager, ape, stranger):
-    rewards_manager.transfer_ownership(stranger, {"from": ape})
+    tx = rewards_manager.transfer_ownership(stranger, {"from": ape})
 
     assert rewards_manager.owner() == stranger
+    assert len(tx.events) == 1
+    assert tx.events[0]['new_owner'] == stranger
```

Comment:
It's better to check events by name via `tx.events['OwnershipTransferred']['new_owner']`. Also in this test case check for `previous_owner` is missing.